src/components/input/Index.module.css

```diff
@@ -27,3 +27,7 @@
 .input:focus {
   border-color: rgba(21, 93, 251, 1);
 }
+
+.errorInput {
```

Comment:
These classed are isolated and only being used for the input component, so no need to use input to affix these classes. I would just call is `.error`